### PR TITLE
[4.0] Add parameters to hide "no contacts" message and "subcategories" title in Contacts

### PR DESCRIPTION
**Pull request by drmenzelit**
Pull Request for Issue #31614 .

### Summary of Changes
Add parameters to hide "no contacts" message and "subcategories" title in 
Contacts » List Contacts in a Category and 
Contacts » List All Contact Categories

### Testing Instructions
Create a category and some subcategories in components -> contacts
Create a menu item of type
Contacts » List Contacts in a Category and / or
Contacts » List All Contact Categories

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
You will find two new parameters in the menu item and also in the global options for Contacts
grafik

grafik

The "no contacts" message and the "subcategories" title can be hidden
grafik

### CODE REVIEW

**Comment by brianteeman** [comment]
Should there be `validate="options"` or even `filter="boolean"` ?

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on f389d3877e177517f77e79e17802a3f4957077fb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31615">issues.joomla.org/tracker/joomla-cms/31615</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f389d3877e177517f77e79e17802a3f4957077fb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31615">issues.joomla.org/tracker/joomla-cms/31615</a>.</sub>

File: components/com_contact/tmpl/category/default_items.php [hunk 1]
```diff
@@ -61,10 +61,12 @@ class="inputbox" onchange="document.adminForm.submit();"
 		<?php endif; ?>
 
 		<?php if (empty($this->items)) : ?>
-			<div class="alert alert-info">
-				<span class="icon-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
+			<?php if ($this->params->get('show_no_contacts', 1)) : ?>
+				<div class="alert alert-info">
+					<span class="icon-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
```
Comment (Quy) [inline components/com_contact/tmpl/category/default_items.php:66]:
```suggestion
					<span class="icon-info-circle" aria-hidden="true"></span><span class="visually-hidden"><?php echo Text::_('INFO'); ?></span>
```

**Comment by drmenzelit** [comment]
Closing this in favour of #32736 due to branch conflicts

**Pull request closed without merging**